hack/tools.go

```diff
@@ -23,4 +23,6 @@ import (
 	_ "k8s.io/code-generator"
 	_ "k8s.io/code-generator/cmd/deepcopy-gen"
 	_ "knative.dev/serving/test/test_images/grpc-ping"
+	_ "knative.dev/serving/test/test_images/multicontainer/servingcontainer"
+	_ "knative.dev/serving/test/test_images/multicontainer/sidecarcontainer"
```

Comment:
just curious, why is this needed ?

Reply:
Those are used in e2e tests. There's `./test/test-images/multicontainer/release/yaml` thats points to that code.